src/components/Menu/src/menu.css

```diff
@@ -0,0 +1,3 @@
+.denhaag-menu :not(:last-child) {
+      margin-inline-end: var(--denhaag-menu-button-margin-inline-end);
+  }
```

Comment:
And I still strongly prefer using `column-gap` since it's much more explicit about what we're doing, instead of adding a margin through a :not selector.